### PR TITLE
Enable UnusedUse sniff again

### PR DESCRIPTION
**Pull request by derrabus**
Keeping the UnusedUse sniff deactivated entirely does more harm than good, I figured. The sniff is still not perfect when it comes to annotations, but if we keep annotations multiline, it works. I think, we can live with that.

### CODE REVIEW

File: tests/Doctrine/Tests/Models/DDC117/DDC117Article.php [hunk 1]
```diff
@@ -20,7 +20,8 @@ class DDC117Article
 {
     /**
      * @var int
-     * @Id @Column(type="integer", name="article_id") @GeneratedValue
+     * @Id @Column(type="integer", name="article_id")
```
Comment (greg0ire) [inline tests/Doctrine/Tests/Models/DDC117/DDC117Article.php]: Is it deliberate to have this one stay single line?

**Comment by beberlei** [comment]
Couldn't we ignore tests/ for this sniff only if it causes these problems with annotations?

File: tests/Doctrine/Tests/Models/DDC117/DDC117Article.php (hunk 1 above)
Comment (derrabus) [reply tests/Doctrine/Tests/Models/DDC117/DDC117Article.php]: No, adjusted. 👍🏻

**Comment by derrabus** [comment]
> Couldn't we ignore tests/ for this sniff only if it causes these problems with annotations?

We could, but why should we? Our test suite does not need to cover all edge cases of the `doctrine/annotations` library, so adhering to a syntax that CodeSniffer understands seems okay to me. I think we gain more from a rule that removes unused imports from the tests, especially if at the same time we require code to explicitly import elements from the global scope.

**Review by greg0ire — APPROVED** [review]
